src/components/Trending/index.js

```diff
@@ -1,11 +1,15 @@
 import React, { Component } from 'react';
 import { Row } from 'reactstrap';
+import logo from '../../logo.png';
+import { Link } from 'react-router-dom';
 
 class Trending extends Component {
   render() {
     return (
       <Row className="flex-grow-1 flex-shrink-1">
-        <h2>Trending page</h2>
+        <img src={logo} size="100px" />
+        <Link to="/explore/">Explore</Link>
```

Comment:
Great, I'll update the `navbar` to include the proper navigation later on tonight.